src/Microsoft.DotNet.Build.Tasks/Targets/resources.targets

```diff
@@ -27,12 +27,21 @@
         ResxFilePath="$(StringResourcesPath)" 
         IntermediateFilePath="$(IntermediateResOutputFileFullPath)" 
         OutputSourceFilePath="$(ResourceSRPath)" 
-        AssemblyName="$(AssemblyName)" >
+        AssemblyName="$(AssemblyName)" 
+        ContinueOnError="ErrorAndStop" >
+        
+        <Output PropertyName="GenerateResourcesCodeOutputMessage" TaskParameter="Message" />
```

Comment:
I wouldn't expect you to need to pick up the message as an out parameter now that you are calling Log.Error I would expect it to already do what you appear to be doing with that message.